<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashboardWidget</class>
 <widget class="QWidget" name="DashboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>800</height>
   </rect>
  </property>
  <property name="objectName">
   <string>dashboard_widget</string>
  </property>
  <layout class="QVBoxLayout" name="dashboard_layout">
   <property name="spacing">
    <number>32</number>
   </property>
   <property name="leftMargin">
    <number>32</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>32</number>
   </property>
   <property name="bottomMargin">
    <number>32</number>
   </property>
   <item>
    <!-- Welcome Section -->
    <widget class="QWidget" name="welcome_section">
     <property name="objectName">
      <string>welcome_section</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="welcome_layout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="welcome_text">
        <layout class="QVBoxLayout" name="welcome_text_layout">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="welcome_title">
           <property name="text">
            <string>Bienvenido de vuelta, Dr. M√©dico</string>
           </property>
           <property name="objectName">
            <string>welcome_title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="welcome_subtitle">
           <property name="text">
            <string>Aqu√≠ tienes un resumen de la actividad de hoy</string>
           </property>
           <property name="objectName">
            <string>welcome_subtitle</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="welcome_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_new_evaluation">
        <property name="text">
         <string>+ Nueva Evaluaci√≥n VNG</string>
        </property>
        <property name="objectName">
         <string>btn_new_evaluation</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <!-- Stats Cards Row -->
    <widget class="QWidget" name="stats_section">
     <property name="objectName">
      <string>stats_section</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>140</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="stats_layout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <!-- Patients Card -->
       <widget class="QWidget" name="stats_patients">
        <property name="objectName">
         <string>stats_patients</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>120</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="patients_card_layout">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>24</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="patients_label">
           <property name="text">
            <string>üë• PACIENTES TOTALES</string>
           </property>
           <property name="objectName">
            <string>patients_label</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="patients_value">
           <property name="text">
            <string>1,247</string>
           </property>
           <property name="objectName">
            <string>patients_value</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <!-- Evaluations Card -->
       <widget class="QWidget" name="stats_evaluations">
        <property name="objectName">
         <string>stats_evaluations</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>120</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="evaluations_card_layout">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>24</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="evaluations_label">
           <property name="text">
            <string>üëÅÔ∏è EVALUACIONES VNG</string>
           </property>
           <property name="objectName">
            <string>evaluations_label</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="evaluations_value">
           <property name="text">
            <string>89</string>
           </property>
           <property name="objectName">
            <string>evaluations_value</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <!-- Reports Card -->
       <widget class="QWidget" name="stats_reports">
        <property name="objectName">
         <string>stats_reports</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>120</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="reports_card_layout">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>24</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="reports_label">
           <property name="text">
            <string>üìÑ REPORTES GENERADOS</string>
           </property>
           <property name="objectName">
            <string>reports_label</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="reports_value">
           <property name="text">
            <string>156</string>
           </property>
           <property name="objectName">
            <string>reports_value</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <!-- Success Rate Card -->
       <widget class="QWidget" name="stats_success">
        <property name="objectName">
         <string>stats_success</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>120</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="success_card_layout">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>24</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="success_label">
           <property name="text">
            <string>‚úÖ TASA DE √âXITO</string>
           </property>
           <property name="objectName">
            <string>success_label</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="success_value">
           <property name="text">
            <string>94.2%</string>
           </property>
           <property name="objectName">
            <string>success_value</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <!-- Main Content Row -->
    <widget class="QWidget" name="content_section">
     <property name="objectName">
      <string>content_section</string>
     </property>
     <layout class="QHBoxLayout" name="content_layout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <!-- Recent Patients -->
       <widget class="QWidget" name="recent_patients">
        <property name="objectName">
         <string>recent_patients</string>
        </property>
        <layout class="QVBoxLayout" name="recent_patients_layout">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>24</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="recent_patients_title">
           <property name="text">
            <string>Pacientes Recientes</string>
           </property>
           <property name="objectName">
            <string>recent_patients_title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="recent_patients_list">
           <property name="objectName">
            <string>recent_patients_list</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>300</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Mar√≠a Gonz√°lez - 45 a√±os - VNG Completa</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Carlos Rodr√≠guez - 52 a√±os - Evaluaci√≥n B√°sica</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ana Mart√≠nez - 38 a√±os - VNG Completa</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Luis Herrera - 61 a√±os - Seguimiento</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Carmen L√≥pez - 29 a√±os - Primera Consulta</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_view_all_patients">
           <property name="text">
            <string>Ver Todos los Pacientes</string>
           </property>
           <property name="objectName">
            <string>btn_view_all_patients</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <!-- Quick Actions -->
       <widget class="QWidget" name="quick_actions">
        <property name="objectName">
         <string>quick_actions</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="actions_layout">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>24</number>
         </property>
         <property name="topMargin">
          <number>24</number>
         </property>
         <property name="rightMargin">
          <number>24</number>
         </property>
         <property name="bottomMargin">
          <number>24</number>
         </property>
         <item>
          <widget class="QLabel" name="actions_title">
           <property name="text">
            <string>Acciones R√°pidas</string>
           </property>
           <property name="objectName">
            <string>actions_title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_new_patient">
           <property name="text">
            <string>üë§ Nuevo Paciente</string>
           </property>
           <property name="objectName">
            <string>btn_new_patient</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_start_vng">
           <property name="text">
            <string>üëÅÔ∏è Iniciar Evaluaci√≥n VNG</string>
           </property>
           <property name="objectName">
            <string>btn_start_vng</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_generate_report">
           <property name="text">
            <string>üìÑ Generar Reporte</string>
           </property>
           <property name="objectName">
            <string>btn_generate_report</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_view_calendar">
           <property name="text">
            <string>üìÖ Ver Calendario</string>
           </property>
           <property name="objectName">
            <string>btn_view_calendar</string>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="actions_spacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <!-- System Status -->
          <widget class="QWidget" name="system_status">
           <property name="objectName">
            <string>system_status</string>
           </property>
           <layout class="QVBoxLayout" name="status_layout">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="status_title">
              <property name="text">
               <string>Estado del Sistema</string>
              </property>
              <property name="objectName">
               <string>status_title</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="status_camera">
              <property name="text">
               <string>üü¢ C√°mara: Conectada</string>
              </property>
              <property name="objectName">
               <string>status_camera</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="status_database">
              <property name="text">
               <string>üü¢ Base de Datos: Activa</string>
              </property>
              <property name="objectName">
               <string>status_database</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="status_backup">
              <property name="text">
               <string>üü° √öltimo Backup: Hace 2 horas</string>
              </property>
              <property name="objectName">
               <string>status_backup</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>